app/controllers/sign_in_tokens_controller.rb

```diff
@@ -45,6 +45,13 @@ def create
 
     if @sign_in_token_form.email_is_user?
       token = SessionService.send_magic_link_email!(@sign_in_token_form.email_address)
+
+      if ENV['HEROKU'] == 'heroku'
```

Comment:
Could this be a `FeatureFlag` that spells out the behaviour it governs,  rather than a reference to a hosting environment? 
Something like `FeatureFlag.active?(:display_sign_in_token_links)` or similar?

Reply:
Feature flag sounds good and I like the idea of using a config setting, but one thing that is concerning me with using that approach here is that we don't want someone to accidentally enable the feature flag and expose the whole service. So maybe it shouldn't be seen as another feature flag and the dev should go out of their way to set it?

- if we want to use config settings perhaps it should be something more generic so that we can reuse it for other features we might want to only be allowed to use/see in review apps. Maybe something along the lines of `FeatureFlag.active?(:review_app_enabled)`?

Reply:
So I think both your comments are polar opposites. I'm with Mr. Davidson on this one, as I think one is less likely to make a mistake if the intent is explicit and direct as opposed to vague and indirect.

Reply:
I've added this as a new commit. it now uses the `FeatureFlag` service and explicitly named. I've also added some coverage for this